src/Hng.Domain/Entities/Organization.cs

```diff
@@ -27,6 +27,7 @@ public class Organization : EntityBase
     public Guid OwnerId { get; set; }
 
     public ICollection<User> Users { get; set; } = new List<User>();
+    public ICollection<UserRole> UsersRoles { get; set; } = new List<UserRole>();
```

Comment:
There is no point having this here. Once you add a collection of Roles in User and collection of Users in Role, EntityFramework creates the link table.

Reply:
organisations need to know their users roles